<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	text-align: center;
	color: rgb(255,255,255);
	font: 75 16pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>OBS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	text-align: center;
	color: rgb(255,255,255);
	font-size: 16px;
	font: 75 16pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>Tamanho</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="quantidade">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox{
	border-radius: 4px;
	background-color: rgb(225, 225, 225);
	border: 1px solid 
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	text-align: center;
	color: rgb(255,255,255);
	font-size: 9px;
	font: 75 16pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>Quantidade</string>
    </property>
   </widget>
   <widget class="QComboBox" name="borda">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	border-radius: 4px;
	background-color: rgb(225, 225, 225);
	border: 1px solid 
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	text-align: center;
	color: rgb(255,255,255);
	font-size: 16px;
	font: 75 16pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>Borda</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="styleSheet">
        <string notr="true">QComboBox{
	border-radius: 4px;
	background-color: rgb(225, 225, 225);
	border: 1px solid 
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_5">
       <property name="styleSheet">
        <string notr="true">QComboBox{
	border-radius: 4px;
	background-color: rgb(225, 225, 225);
	border: 1px solid 
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_3">
       <property name="styleSheet">
        <string notr="true">QComboBox{
	border-radius: 4px;
	background-color: rgb(225, 225, 225);
	border: 1px solid 
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">QLabel{
	text-align: center;
	color: rgb(255,255,255);
	font-size: 16px;
	font: 75 16pt &quot;MS Shell Dlg 2&quot;;
}</string>
       </property>
       <property name="text">
        <string>Sabores</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="tamanho">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	border-radius: 4px;
	background-color: rgb(225, 225, 225);
	border: 1px solid 
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finalizar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>560</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
    </property>
    <property name="text">
     <string>Finalizar Pedido</string>
    </property>
   </widget>
   <widget class="QLabel" name="fundo">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-30</y>
      <width>491</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/pizza-g0b57d8a44_1920.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>fundo</zorder>
   <zorder>textEdit</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>quantidade</zorder>
   <zorder>label_3</zorder>
   <zorder>borda</zorder>
   <zorder>label_4</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>tamanho</zorder>
   <zorder>finalizar</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
